user_manual/groupware/sync_android.rst

```diff
@@ -18,6 +18,11 @@ Files and notifications
    **Security**. Generate an App password, click "Generate QR code" and
    tap the QR scanner icon in the Nextcloud app, point your phone's
    camera towards the screen.
+3. Once you are signed in you can view and edit files from within the app similar to how you would view and edit the files from within a browser. 
```

Comment:
```suggestion
3. Once you are signed in, you can view and edit files from within the app similar to how you would view and edit the files from within a browser. 
```

Reply:
I would not compare it with a browser but with a local file explorer.